DOCS/man/options.rst

```diff
@@ -2906,11 +2906,6 @@ Input
 
     (This option was renamed from ``--input-x11-keyboard``.)
 
-``--input-app-events=<yes|no>``
-    (OS X only)
-    Enable/disable application wide keyboard events so that keyboard shortcuts
-    can be processed without a window. Enabled by default (except for libmpv).
-
```

Comment:
Removal should go into DOCS/interface-changes.rst too. I assume nobody really uses this option explicitly, so we probably don't need any deprecation or such?